docs/rules/object-shorthand.md

```diff
@@ -100,6 +100,37 @@ You can set the option in configuration like this:
 }
 ```
 
+While set to `"always"`, `"methods"`, or `"properties"`, shorthand syntax using string literal keys can be ignored using the optional parameter `"avoidQuotes"`. This will prevent warnings for properties with string keys, even if the name of the keys exactly match the name of the reference. It will also prevent warnings on longform syntax for methods with string keys, and will warn for shorthand method syntax where the key is a string literal. Note: The first parameter must be specified when using this optional parameter.
```

Comment:
> This will prevent warnings for properties with string keys, even if the name of the keys exactly match the name of the reference. It will also prevent warnings on longform syntax for methods with string keys, and will warn for shorthand method syntax where the key is a string literal.

FWIW it was a bit hard to understand this until I read the code examples below. Perhaps we could try to keep this paragraph shorter and give more emphasis to the code examples? Maybe:

> "This will make it so longform syntax is preferred when the object key is a string literal."


Just a suggestion :)